### PR TITLE
Add docs for 0.13.1 fleet rules package

### PR DESCRIPTION
**Pull request by brokensound77**
As of 7.13, rules can now be released out of band (OOB) via a fleet integration. This will add some new structure to the docs for prebuilt rules to incorporate those changes.

<details>
<summary>Intended layout</summary><br>

image
</details>

----

*As a side note, there were some flaws noticed in how rule docs have been generated since inception and so the intent is to rebuild them all from initial release, but this will come in a separate PR. This will also allow for a better way to include these OOB changes, so for now, these changes included here should be temporary*

[preview](https://security-docs_769.docs-preview.app.elstc.co/guide/en/security/master/prebuilt-rule-0-13-1-prebuilt-rules-0-13-1-summary.html)

### CODE REVIEW

**Comment by nastasha-solomon** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

File: docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+[role="xpack"]
+== Downloadable updates
+
+This section lists all updates to prebuilt rules released via the detection rules fleet integration.
+
+
+
+[width="100%",options="header"]
+|==============================================
+|Release |Date | New rules | Updated rules
```
Comment (rw-access) [inline docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc:11]:
```suggestion
| Update version | Date | New rules | Updated rules
```

File: docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[[prebuilt-rules-downloadable-updates]]
+[role="xpack"]
+== Downloadable updates
+
+This section lists all updates to prebuilt rules released via the detection rules fleet integration.
```
[the remark this answers is not in the document]
Comment (rw-access) [reply docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc:5]: I don't know if this needs an `include::` or something else to work.

File: docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+
+
+[width="100%",options="header"]
+|==============================================
+|Release |Date | New rules | Updated rules
+
+|<<prebuilt-rule-0-13-1-prebuilt-rules-0-13-1-summary, 0.13.1>> | 17 Jun 2021 | 4 | 41
```
Comment (rw-access) [inline docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc:13]: assuming it's today. I think we should go off of UTC for this
```suggestion
|<<prebuilt-rule-0-13-1-prebuilt-rules-0-13-1-summary, 0.13.1>> | 21 June 2021 | 4 | 41
```

File: docs/detections/prebuilt-rules/downloadable-packages/0-13-1/prebuilt-rules-0-13-1-summary.asciidoc [hunk 4]
```diff
@@ -0,0 +1,150 @@
+[[prebuilt-rule-0-13-1-prebuilt-rules-0-13-1-summary]]
+[role="xpack"]
+== Prebuilt rules updates for downloadable package 0.13.1
+
+This section lists all updates to prebuilt rules released via the detection rules fleet integration.
```
Comment (rw-access) [inline docs/detections/prebuilt-rules/downloadable-packages/0-13-1/prebuilt-rules-0-13-1-summary.asciidoc:5]:
```suggestion
This section lists all updates associated with version 0.13.1 of the Fleet integration *Prebuilt Security Detection Rules*.
```

**Review by rw-access — COMMENTED** [review]
looks pretty good, just need the 🟢 and some verbiage changes

File: docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc [hunk 5]
```diff
@@ -0,0 +1,18 @@
+[[prebuilt-rules-downloadable-updates]]
+[role="xpack"]
+== Downloadable updates
```
Comment (rw-access) [inline docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc:3]:
```suggestion
== Downloadable rule updates
```

File: docs/detections/prebuilt-rules/downloadable-packages/0-13-1/prebuilt-rules-0-13-1-summary.asciidoc [hunk 6]
```diff
@@ -0,0 +1,150 @@
+[[prebuilt-rule-0-13-1-prebuilt-rules-0-13-1-summary]]
+[role="xpack"]
+== Prebuilt rules updates for downloadable package 0.13.1
```
Comment (rw-access) [inline docs/detections/prebuilt-rules/downloadable-packages/0-13-1/prebuilt-rules-0-13-1-summary.asciidoc:3]:
```suggestion
== Prebuilt Security Detection Rules update v0.13.0
```
also, I noticed that this isn't rendering in the TOC properly. ideas?
Screen Shot 2021-06-22 at 12 35 02 PM

File: docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+[width="100%",options="header"]
+|==============================================
+|Update version |Date | New rules | Updated rules
+
+|<<prebuilt-rule-0-13-1-prebuilt-rules-0-13-1-summary, 0.13.1>> | 21 Jun 2021 | 4 | 41
+
+|==============================================
+
+
+include::downloadable-packages/0-13-1/prebuilt-rules-0-13-1-summary.asciidoc[]
```
Comment (lcawl) [inline docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc:19]: If you want to bump pages down a level in the navigation tree (without having to edit the === heading level in each one), you can use the [leveloffset attribute](https://docs.asciidoctor.org/asciidoc/latest/directives/include-with-leveloffset/#manipulate-heading-levels-with-leveloffset) like this:
```suggestion
include::downloadable-packages/0-13-1/prebuilt-rules-0-13-1-summary.asciidoc[leveloffset=+1]
```

Comment (brokensound77) [reply docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc:19]: ok perfect - this may be exactly what I need, thanks!

**Review by rw-access — APPROVED** [review]
LGTM 🚢

**Pull request merged**